dataaccess-couch.js

```diff
@@ -28,22 +29,21 @@ var allPlayers = {};
 
 function statsInitialized() {
     return new Promise(function (resolve) {
-        (function waitForInitialStatsToBeGenerated(){
+        setTimeout(() => {
```

Comment:
The original function ran repeatedly until `stats != null`, now it only runs once.